spatialos/UnrealGDK
# Bugfix/unr 2501 tombstone check to log

**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
Convert Tombstone-related assert to log to prevent editor crashes in the case it fires.

#### Release note
Added to changelog.md.

#### Tests
Verified tombstone functionality still works as expected through PIE as follows:

1. Disable "delete dynamically spawned entities"
2. Author small test level containing object that can be destroyed
3. Start Spatial + PIE session
4. Destroy object
5. Quit PIE
6. Restart PIE
7. Observe object remains deleted

STRONGLY SUGGESTED: How can this be verified by QA?
As described under tests above

#### Documentation
changelog.md

#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them.
